lib/pure/times.nim

```diff
@@ -276,8 +275,7 @@ type
     dSun = "Sunday"
 
 type
-  MonthdayRange* = range[0..31]
-    ## 0 represents an invalid day of the month
+  MonthdayRange* = range[1..31]
   HourRange* = range[0..23]
   MinuteRange* = range[0..59]
   SecondRange* = range[0..60]
```

Comment:
pre-existing but maybe add a comment saying that the range is 0..60 to account for leap seconds ? (I'm not even sure that's the reason, but it sure looks weird otherwise)

Reply:
I added a comment. There's not really any good reason that the range includes `60`, `DateTime` will never contain a leap second with the current implementation. The only difference it makes is that `initDateTime` allows timestamps with leap seconds

Reply:
> The only difference it makes is that initDateTime allows timestamps with leap seconds

I guess that's a good enough reason for existence? if possible you could add a test that calls initDateTime with a leap second, unless it's too much work because it would fail validation as currently implemented